### PR TITLE
More issue fixes

### LINKED ISSUE #218: Deleting instrument
When deleting an instrument from the project, it is not deleted from any shots.

### LINKED ISSUE #219: Instruments with the same name
When you have two instruments with the same name, some things go wrong in the dropdown menu

### PR DESCRIPTION
**Pull request by mennoo1996**
Fixes #218 
Fixes #219

### CODE REVIEW

File: src/main/java/control/DirectorTimelineController.java [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
 import data.GeneralShotData;
+import data.Instrument;
 import data.Shot;
 import gui.centerarea.DirectorShotBlock;
+import gui.centerarea.ShotBlock;
 import gui.events.DirectorShotBlockUpdatedEvent;
 import gui.root.RootPane;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.extern.log4j.Log4j2;
```
Comment (bart-de-jonge) [inline src/main/java/control/DirectorTimelineController.java:22]: whitespace

File: src/main/java/control/TimelineController.java [hunk 2]
```diff
@@ -76,6 +77,20 @@ public void addCameraShot(int cameraIndex, CameraShot newShot) {
...
+     * @param instrument the instrument to remove
+     */
+    public void removeInstrumentFromAllShots(Instrument instrument) {
+        this.cameraShotBlocks.forEach(shotBlock -> {
+                shotBlock.getInstruments().remove(instrument);
+                shotBlock.getShot().getInstruments().remove(instrument);
+                shotBlock.getTimetableBlock().removeInstrument(instrument);
+                shotBlock.recompute();
+            });
+
```
Comment (bart-de-jonge) [inline src/main/java/control/TimelineController.java]: whitespace

**Comment by bart-de-jonge** [comment]
Two small comments, other than that looks fine

**Comment by martijnjanssen** [comment]
Looks good.

**Pull request merged**